src/content/guides/multi-nginx/index.md

```diff
@@ -5,20 +5,56 @@ type: page
+  - How do I separate internal and external Services?
+  - How do I configure NGINX Ingress Controller for internal traffic?
+  - How do I override the NodePorts on KVM Ingresses?
+  - How do I configure NGINX Ingress Controller to allow weak ciphers for legacy devices?
 ---
 
 # Running Multiple NGINX Ingress Controllers
 
-NGINX ingress controller handles [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources, routing the traffic from outside of the Kubernetes cluster to services within the cluster. Starting with App version 1.8.0 one can install multiple NGINX ingress controllers in same Kubernetes cluster. Here we cover common cases where having multiple NGINX ingress controllers can be useful.
+NGINX ingress controller handles [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources, routing traffic from outside the Kubernetes cluster to services within the cluster.
 
-Most of the NGINX configuration options have NGINX wide defaults but can also be overriden on per Ingress resource level. In each of the covered multi-NGINX use cases, one essentially installs second NGINX with different NGINX global-only configuration, i.e. configuration that applies to all Ingress resources managed by that NGINX installation which cannot be customized on per-Ingress resource level.
+Starting with [NGINX IC v1.8.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#180---2020-07-24), one can install multiple NGINX ingress controllers in a Kubernetes cluster. The optional NGINX Ingress Controller can be [installed as as an App on your cluster]({{< relref "/content/guides/installing-optional-ingress-controller/index.md" >}}). This Ingress Controller is registered with the default `nginx` Ingress Class.
+
+Some use cases for this might be:
+
+- An Ingress Controller that is behind an internal ELB for traffic between services within the VPC (or a group of peered VPCs)
+- An Ingress Controller behind an ELB that already terminates SSL
+- An Ingress Controller with different functionality or performance
+
+__Note__ that if you are running multiple Ingress Controllers you need to annotate each Ingress with the appropriate class, e.g.
+
+```yaml
+kubernetes.io/ingress.class: "nginx"
+```
+
+or
+
+```yaml
+kubernetes.io/ingress.class: "nginx-internal"
+```
+
+Not specifying the annotation will lead to multiple ingress controllers claiming the same ingress. Specifying a value which does not match the class of any existing ingress controllers will result in all ingress controllers ignoring the ingress.
+
+Further note that if you are running additional Ingress Controllers you might need to configure them so their Ingress Class does not collide with the class of our default NGINX Ingress Controller. For the community supported NGINX Ingress Controller this is described in the [official documentation](https://kubernetes.github.io/ingress-nginx/user-guide/multiple-ingress/).
```

Comment:
We no longer have default (preinstalled) nginx in most clusters https://docs.giantswarm.io/general/releases/#apps - how does that change this?

Maybe sth like this?
```suggestion
Additionally, please ensure the Ingress Class of each of your Ingress Controllers do not collide with each other. For the community supported NGINX Ingress Controller this is described in the [official documentation](https://kubernetes.github.io/ingress-nginx/user-guide/multiple-ingress/).
```